src/win/util.c

```diff
@@ -1491,7 +1492,9 @@ int uv_os_getenv(const char* name, char* buffer, size_t* size) {
     if (r == ERROR_ENVVAR_NOT_FOUND)
       return UV_ENOENT;
 
-    return uv_translate_sys_error(r);
+    /* Handle defined, but empty environment variables. */
+    if (r != ERROR_SUCCESS)
```

Comment:
Maybe this condition can be folded with the one above?

Reply:
Do you mean just make this an `else if`?

Reply:
I mean, if `uv_translate_sys_error` would know how to translate `ERROR_ENVVAR_NOT_FOUND` then anything `!= ERROR_SUCCESS` would just be `return uv_translate_sys_error(r);`

Reply:
I like it 👍